README.md

```diff
@@ -1,2 +1,48 @@
-# brein-api-library-react
-React Library for the Breinify API
+# Breinify’s React API Library
+
+<!-- TOC -->
+* [Overview](#overview)
+* [Getting Started](#getting-started)
+  * [Retrieving an API-Key](#retrieving-an-api-key)
+  * [Installing](#installing)
+  * [Configuring the Library](#configuring-the-library)
+* [API](#api)
+* [Further Links](#further-links)
+<!-- TOC -->
+
+## Overview
+Breinify’s React API library provides hooks, components, and type definitions to call Breinify’s API.
+
+## Getting Started
+### Retrieving an API-Key
+You need a valid API-key, which you can get for free at  [https://www.breinify.com](https://www.breinify.com/) . In the examples, we assume you have the following API key:
+**938D-3120-64DD-413F-BB55-6573-90CE-473A**
+It is recommended to use signed messages when utilizing the react library. A signed messages ensures, that the request is authorized. To activate signed message ensure that `Verification Signature` is enabled for your key (see  [Breinify’s API Docs](https://docs.breinify.com/#request-an-api-key)  for further information). In this documentation we assume that the following secret is attached to the API key and used to sign a message.
+**utakxp7sm6weo5gvk7cytw==**
+
+### Installing
+The library is available on  [npm](https://www.npmjs.com/package/breinify-node)  and can be added using:
+```bash
+npm i brein-react-api-library-react --save
```

Comment:
We should check if its a dev-dependency or an actual dependency. I'm not sure at this point and time, but something to keep in mind.